src/pages/YourTurn/YourTurn.tsx

```diff
@@ -26,17 +26,7 @@ function buildConfetti() {
   });
 }
 
-export function YourTurn() {
-  const players = [
-    { name: 'Fulano da Silva', id: '1234', gender: 'male' as Gender },
-    {
-      name: 'Beltrano dos Santos Silva',
-      id: '1214',
-      gender: 'male' as Gender,
-    },
-    { name: 'Ciclano de Souza', id: '1224', gender: 'male' as Gender },
-  ];
-
+export function YourTurn({ game = [] }: { game?: Athlete[] }) {
```

Comment:
How about adding a type file for it

```suggestion
type YourTurnProps = { game?: Athlete[] }

export function YourTurn({ game = [] }: YourTurnProps) {
```